packages/tools/src/tools/CrosshairsTool.ts

```diff
@@ -367,8 +367,8 @@ class CrosshairsTool extends AnnotationTool {
    * @returns Crosshairs annotation
    */
   addNewAnnotation = (
-    evt: EventTypes.MouseDownActivateEventType,
-    interactionType: string
+    evt: EventTypes.InteractionEventType,
+    interactionType = 'mouse'
```

Comment:
I don't think this is used in this function. Minor nit that might be better to do in a separate PR.